paloalto_wildfire/komand_paloalto_wildfire/actions/get_verdict/action.py

```diff
@@ -16,7 +16,7 @@ def run(self, params={}):
         client = self.connection.client
         out = client.get_verdicts(params.get('hash'))
 
-        return { 'verdict': out }
+        return { 'verdict': out.capitalize() }
```

Comment:
```suggestion
        return { 'verdict': out.capitalize() }
```

Does the has need to be capitalized?

Reply:
Yes,  enumerated outputs did not match schema. Here we can avoid a major version bump.

```
        enum:
        - Benign
        - Malware
        - Greyware
        - Pending
        - Error
        - Not found
```